php/deepgram-Audio-Summary/index.php

```diff
@@ -0,0 +1,20 @@
+<?php
+ 
+function deepgramAudioSummary($fileUrl)
+{
+    $secretKey = "Your-Secret-Key";
+    $request = "https://api.deepgram.com/v2/speech?fileUrl=" + $fileUrl;
+    $ch = curl_init($request);
+ 
+    curl_setopt($ch, CURLOPT_HTTPHEADER, array(
+        "Content-Type: application/json",
+        "Authorization: Bearer " + $secretKey
+    ));
+ 
+    $response = curl_exec($ch);
+    curl_close($ch);
+ 
+    return $response;
+}
+ 
+?>
```

Comment:
This is not the correct way to implement a PHP Open Runtimes Function. Please see how the other examples are structured and be sure to test the function like documented in the readme.